<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionDialog</class>
 <widget class="QDialog" name="QuestionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>A</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>B</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pushButton_4">
     <property name="text">
      <string>D</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_1">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>C</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButton_5">
     <property name="text">
      <string>Neste Spørsmål</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton_6">
     <property name="text">
      <string>Avslutt test</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
